buffer-env.el

```diff
@@ -140,10 +140,10 @@ Files marked as safe to execute are permanently stored in
 
 (defun buffer-env--locate-script ()
   "Locate a dominating file named `buffer-env-script-name'."
-  (when-let* ((dir (and (stringp buffer-env-script-name)
-                        (not (file-remote-p default-directory))
-                        (locate-dominating-file default-directory
-                                                buffer-env-script-name))))
+  (when-let* (((stringp buffer-env-script-name))
+	      ((not (file-remote-p default-directory)))
+	      (dir (locate-dominating-file default-directory
+					   buffer-env-script-name)))
```

Comment:
My old code was certainly ugly, but your version is equally ugly in my estimation :sweat_smile:. Also, it seems there are some tabs there.